src/planwise/model/import_job.clj

```diff
@@ -390,6 +390,10 @@
                          :type-field type-field)]
     (import-job job-value)))
 
+(defn restore-job
+  [job]
+  (import-job (:state job) (:value job)))
```

Comment:
How does this cope with pending tasks, ie. tasks that were pending at the moment the job was serialized? After restoration, they won't get a succeeded/failed report, because they effectively don't exist, so the FSM can be left in a stuck state.